src/script/lua_api/l_env.h

```diff
@@ -22,6 +22,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "lua_api/l_base.h"
 #include "serverenvironment.h"
 #include "raycast.h"
+#include "mapgen/treegen.h"
```

Comment:
forward declaration should work here without any extra changes

Reply:
Completely removed the include.